packages/js/components/src/product-section-layout/style.scss

```diff
@@ -0,0 +1,52 @@
+.woocommerce-form-section {
+	a {
+		text-decoration: none;
+	}
+
+	&__content {
+		.components-card {
+			border: 1px solid $gray-400;
+			border-radius: 2px;
+			box-shadow: none;
+			&__body {
+				padding: $gap-large;
+
+				> .components-base-control,
+				> .components-dropdown,
+				> .woocommerce-rich-text-editor {
+					&:not(:first-child):not(.components-radio-control) {
+						margin-top: $gap-large - $gap-smaller;
+						margin-bottom: 0;
+					}
+				}
+			}
+		}
+
+		.woocommerce-product-form__field:not(:first-child) {
+			margin-top: $gap-large;
```

Comment:
We may want to change this to the same thing as line 18 -> `margin-top: $gap-large - $gap-smaller;`
I think this css might of been irrelevant prior to this re-structure, but is getting applied now.
The margin between fields should be `16px` as seen in the pricing section, now it uses `24px` which is quite noticeable:
<img width="794" alt="Screenshot 2023-01-16 at 1 26 14 PM" src="https://user-images.githubusercontent.com/2240960/212678326-410dbcfc-a6a1-4f98-8487-788be1bec455.png">

Reply:
Nice catch @louwie17 , remedied in 0f0d875ec14bc6d72577bf9f1d339ca277b4c2e7